### PR TITLE
pmicControl: charger: Added VBatLow enable

### PR DESCRIPTION
**Pull request by aasinclair**
Added VBatLow enable to charger card.
Includes overlay export.

### CODE REVIEW

**Comment by aasinclair** [comment]
Not urgent, should be merged after the next formal release

**Comment by aasinclair** [comment]
@kylebonnici This is good to go, and is compatible with the existing FW

**Review by greg-fer — APPROVED** [review]

**Comment by greg-fer** [comment]
> @greg-fer just have a look at the tool tip text other then that all is good for me :)

Looks good to me. Thanks for bringing me in.

File: src/components/Cards/Power/PowerCard.tsx [hunk 1]
```diff
@@ -146,6 +146,24 @@ export default ({
...
                         disabled={disabled}
                     />
+                    <Toggle
+                        label={
+                            <DocumentationTooltip
+                                card={card}
+                                item="EnableVBatLow"
+                            >
+                                <>
+                                    <span>Enable Charging Below V</span>
```
Comment (greg-fer) [inline src/components/Cards/Power/PowerCard.tsx:156]: The only suggestion I might have is that this line might be a bit long for UI. So I might suggest something like:
```suggestion
                                    <span>Charging Below V</span>
```
Assuming that there is a toggle below this, we don't need Enable in the option name.

File: src/features/pmicControl/npm/npm1300/documentationPmic1300.tsx [hunk 2]
```diff
@@ -787,6 +787,23 @@ export const documentation: Documentation = {
                 ],
             },
         ],
+        EnableVBatLow: [
+            {
+                title: (
+                    <>
+                        <span>Enable Charging Below V</span>
```
Comment (greg-fer) [inline src/features/pmicControl/npm/npm1300/documentationPmic1300.tsx:794]: See previous comment.

**Pull request merged**